recycler/src/main/java/com/elpassion/android/commons/recycler/adapters/factory.kt

```diff
@@ -1,7 +1,12 @@
 package com.elpassion.android.commons.recycler.adapters
```

Comment:
Please rename this class. It should be more explanatory and it should be convenient to use also in java - use factory.basicAdapterWithHolder may look strange.